pkg/cmd/stack/deploy.go

```diff
@@ -545,7 +545,9 @@ func isAnyPortAvailable(ctx context.Context, svc *model.Service, stack *model.St
 			continue
 		}
 	}
-	forwarder.Start(podName, stack.Namespace)
+	if err := forwarder.Start(podName, stack.Namespace); err != nil {
+		oktetoLog.Infof("could not start port-forward: %s", err)
```

Comment:
@jLopezbarb should we `continue` here as we do with the other errors?

Reply:
Yes, this is just a healthcheck while deploying an stack. We can't fail for a healtcheck and wait until the backofflimit is reached

Reply:
```suggestion
		oktetoLog.Infof("could not start port-forward: %s", err)
		continue
```

@abs007 :eyes: